src/main/java/application/baseball/Game.java

```diff
@@ -0,0 +1,59 @@
+package application.baseball;
+
+import application.constant.Message;
+
+import java.util.List;
+
+public class Game {
+    private Computer computer;
+    private Integer gameStatus; // 0: 게임시작전 1: 게임중 2: 게임종료
+    private Input input;
+    private Judgement judgement;
```

Comment:
```suggestion
    private final Computer computer;
    private final Input input;
    private final Judgement judgement;
    private Integer gameStatus;
```

클래스에 멤버 필드를 초기화 할 때, 생성자에서 초기화가 이루어지는 필드에 대해서는 불변을 유지하기 위한 final 키워드를  붙여야 합니다. 

자바는 불변 가변을 강제하지 않아, 협업을 할 때 특정 필드의 의도가 불변임에도 다른 사람이 코드를 수정할 때 가변처럼 필드를 다른 메서드에서 생성할 가능성이 있기 때문입니다.

기본적으로 자바 객체지향의 SRP를 지키기 위해서는 캡슐화 및 불변을 지키는 것을 목표로 해보세요.!

Reply:
넵! 감사합니다..!

헌데 궁금한 것이 computer라는 객체 내 멤버 필드로 answer라는 리스트 변수가 있습니다.
이 변수에는 새 게임을 시작하게 되면 새로운 수를 generate해야 하는데, 이 때는 값이 변하게 됩니다.
이는 input 객체에도 동일하게 적용되고 있구요!

이런 객체들은 final이 붙어도 내부 필드 값 변경에 제약이 없나요?

Reply:
넵 computer라는 객체 안애 참조를 변경하는 것이라 괜찮습니다.

Reply:
그렇군요.! 답변 감사합니다